airbyte-db/lib/configs_db_schema.md

```diff
@@ -0,0 +1,359 @@
+# Configs Database Schema Draft
+
+This is a temporarily file that documents the design of the new configs database. Currently everything lives in the `airbyte_configs` table. We want to normalize it to one table per config type. Relevant issue: [#5479](https://github.com/airbytehq/airbyte/issues/5479).
+
+## Convention
+- The tables are as similar to the YAML definition as possible.
```

Comment:
Yes. Also to avoid having lots of new or different field names as current schema. I'd like to lower the cognitive cost for us to adapt to the schema.